### DISCUSSION TITLE
When I Close A Connection Or Delete A Queue, It Comes Back

### DISCUSSION THREAD

**Discussion by andreweberle** [rabbitmq/rabbitmq-server#3245] [Other]
Whenever I attempt to close a connection with the following
`.\rabbitmqctl.bat close_all_connections rabbit@name` or with the UI.

All 77 connections are automatically added back.

I tried removing the application all together, however I connections still remain.

I also tried remove all queues first and then the connections
```
$queues = .\rabbitmqctl.bat list_queues name
foreach ($queue in $queues)
{
    .\rabbitmqctl.bat delete_queue $queue
}
```

which said they removed successfully,
but they still come back.

One thing I noticed is if you change the password of the account you're using, the connections will remain deleted,
however if you change it back to the original password, the connections will come back.

Here is the log from when I attempt to remove a single connection
```
2021-07-29 10:53:25.640 [error] <0.21494.1> Error on AMQP connection <0.21494.1> ([fe80::d47b:27dc:26be:d3f9]:54208 -> [fe80::d47b:27dc:26be:d3f9]:5672, vhost: '/', user: 'guest', state: running), channel 0:
 operation none caused a connection exception connection_forced: "Closed via management plugin"
2021-07-29 10:53:25.640 [info] <0.24004.1> Closing connection <0.21494.1> because "Closed via management plugin"
2021-07-29 10:53:25.640 [info] <0.21494.1> closing AMQP connection <0.21494.1> ([fe80::d47b:27dc:26be:d3f9]:54208 -> [fe80::d47b:27dc:26be:d3f9]:5672, vhost: '/', user: 'guest')
2021-07-29 10:53:25.640 [info] <0.24011.1> Closing all channels from connection '[fe80::d47b:27dc:26be:d3f9]:54208 -> [fe80::d47b:27dc:26be:d3f9]:5672' because it has been closed
```

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
My best guess with this much information provided is that your application(s) recover and redeclare. RabbitMQ does not declare
queues (if we ignore plugins such as Shovel and Federation, which are specialised apps co-hosted in the same cluster).

**Comment by michaelklishin** [comment]
…and RabbitMQ most certainly cannot recover any closed client connection. It does not connect to clients, instead, clients connect to RabbitMQ.

**Reply by andreweberle** [reply]
Hey Michael, Turns out there was another user logged into the server which had some connections open.
Thanks again for taking the time to have a look :)
